wagtail/utils/setup.py

```diff
@@ -29,10 +29,11 @@ def bump_client_version(self):
         Writes the current Wagtail version number into package.json
         """
         path = os.path.join(".", "client", "package.json")
-        input_file = open(path)
 
+        package = {}
         try:
-            package = json.loads(input_file.read().decode("utf-8"))
+            with open(path) as f:
```

Comment:
Suggestion: If you open the file in text mode, there's no need to additionally decode the output.

Reply:
It didn't occur to me. Thanks!